Packs/BluelivThreatCompass/pack_metadata.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "Blueliv ThreatCompass",
+    "description": "Blueliv ThreatCompass systematically looks for information about companies,products, people, brands, logos, assets, technology and other information, depending on your needs. Blueliv ThreatCompass allows you to monitor and track all this information to keep your data, your organization and its employees safe",
+    "support": "community",
```

Comment:
@glicht @anara123 @yuvalbenshalom that support type is ok, right ?

Reply:
yes